content/posts/2023-03-05-gather-town-building.md

```diff
@@ -0,0 +1,206 @@
+
+![Interaction_Area](images/2023-03-05-gather-town-building/15-Interaction_Area.png)
+
+---
+
+## Conference room
+
+This year, all talks are pre-recorded and released one week ago to attendees. We created 4 projection booths to let attendees watch the talk videos directly on Gather.Town. We also built 4 speakers’ rooms for each speaker to interact with attendees. *- written by Winnie -*
+
+### Projection Booth
+
+![Projection_Booth_1](images/2023-03-05-gather-town-building/16-Projection_Booth_1.png)
+
+For a better viewing experience and not to create several video objects, we used google slides instead to embed these talk videos into Gather.Town. The attendees can watch the talks directly via the slide objects.
+
+![Projection_Booth_2](images/2023-03-05-gather-town-building/17-Projection_Booth_2.jpg)
+
+![Projection_Booth_3](images/2023-03-05-gather-town-building/18-Projection_Booth_3.png)
+
+> Attendees can choose the preferred one and click to watch the talk.
+>
+### Speakers' Room
+
+![Speaker_Room_1](images/2023-03-05-gather-town-building/19-Speaker_Room_1.png)
+
+After watching talk videos, attendees can walk into the speakers room according to the program schedule. In this area, speakers and attendees can communicate freely and directly. For everyone who got questions, this was the perfect time to approach the speakers.
+
+![Speaker_Room_2](images/2023-03-05-gather-town-building/20-Speaker_Room_2.png)
+
+---
+
+## Sponsor & Community booth
+
+### Sponsor booth
+
+This year, we have a lot of sponsors to support us. And also we provide booths for sponsors to interact with attendees. Our booth is an imitation of Taiwan's shopping street so that foreign attendees can also feel the intense atmosphere of Taiwan. *- written by Allen -*
+
+![Sponsor_Booth](images/2023-03-05-gather-town-building/21-Sponsor_Booth.png)
+
+### Community booth
+
+If the attendees want to interact with communities like KIWI PyCon, Taichung.py, MOPCON, PyCon MY, PYCON HK, PYCON JP, and PyCon PH. They can go to the beach. We designed a marketplace called "台味市集" and put lots of booths for each community. *- written by Pochun -*
+
+![Community_Booth_1](images/2023-03-05-gather-town-building/22-Community_Booth_1.png)
+
+![Community_Booth_2](images/2023-03-05-gather-town-building/23-Community_Booth_2.png)
+
+Also, we have a podcast show called PyCast, held by volunteers.
+We put the episode links at the corners of the space.
+
+![Pycast_1](images/2023-03-05-gather-town-building/24-Pycast_1.png)
+
+![Pycast_2](images/2023-03-05-gather-town-building/25-Pycast_2.png)
+
+---
+
+## Booth Game
+
+### Taiwan-travel
+
+This year, we designed a Monopoly as our main space. Working through the path, you can see many icons in the squares. Each icon square has a challenge. Take the challenges to win the prize.  *- written by Fish -*
+
+![PyCon APAC 2022 Monopoly](images/2023-03-05-gather-town-building/26-Monopoly.jpg)
+
+> PyCon APAC 2022 Monopoly
+
+### Mistery-space
```

Comment:
Minor issue. I think it should be Mystery (filename and post typo)

Reply:
Thanks for checking! I've fixed it at the latest commit. 
Please review again, thank you!